KnockKnock-iOS/Scenes/Feed/FeedWrite/Views/FeedWriteView.swift

```diff
@@ -0,0 +1,249 @@
+      promotionSeperateLineView.heightAnchor.constraint(equalToConstant: Metric.seperatorHeight),
+      promotionSeperateLineView.widthAnchor.constraint(equalTo: self.safeAreaLayoutGuide.widthAnchor, constant: Metric.seperatorWidthMargin),
+
+      shopLabel.topAnchor.constraint(equalTo: self.promotionSeperateLineView.bottomAnchor, constant: Metric.shopTopMargin),
+      shopLabel.leadingAnchor.constraint(equalTo: self.safeAreaLayoutGuide.leadingAnchor, constant: Metric.shopLeadingMargin),
+      shopLabel.trailingAnchor.constraint(equalTo: self.safeAreaLayoutGuide.trailingAnchor, constant: Metric.shopTrailingMargin),
+
+      shopSearchImageView.topAnchor.constraint(equalTo: self.shopLabel.topAnchor, constant: Metric.selectImageTopMargin),
+      shopSearchImageView.trailingAnchor.constraint(equalTo: self.safeAreaLayoutGuide.trailingAnchor, constant: Metric.selectImageTrailingMargin),
+      
+      shopSearchButton.topAnchor.constraint(equalTo: self.shopLabel.topAnchor),
+      shopSearchButton.leadingAnchor.constraint(equalTo: self.shopLabel.leadingAnchor),
+      shopSearchButton.trailingAnchor.constraint(equalTo: self.shopSearchImageView.trailingAnchor),
+      shopSearchButton.heightAnchor.constraint(equalToConstant: Metric.selectButtonHeight),
+
+      shopSearchSeperateLineView.topAnchor.constraint(equalTo: self.shopSearchButton.bottomAnchor),
+      shopSearchSeperateLineView.centerXAnchor.constraint(equalTo: self.centerXAnchor),
+      shopSearchSeperateLineView.heightAnchor.constraint(equalToConstant: Metric.seperatorHeight),
+      shopSearchSeperateLineView.widthAnchor.constraint(equalTo: self.safeAreaLayoutGuide.widthAnchor, constant: Metric.seperatorWidthMargin),
+
+      contentTextView.topAnchor.constraint(equalTo: self.shopSearchSeperateLineView.bottomAnchor, constant: Metric.contentTextViewTopMargin),
+      contentTextView.leadingAnchor.constraint(equalTo: self.safeAreaLayoutGuide.leadingAnchor, constant: Metric.contentTextViewLeadingMargin),
+      contentTextView.trailingAnchor.constraint(equalTo: self.safeAreaLayoutGuide.trailingAnchor, constant: Metric.contentTextViewTrailingMargin),
+      contentTextView.heightAnchor.constraint(equalToConstant: Metric.contentTextViewHeight),
+
+      doneButton.heightAnchor.constraint(equalToConstant: Metric.doneButtonHeight),
+      doneButton.bottomAnchor.constraint(equalTo: self.safeAreaLayoutGuide.bottomAnchor, constant: Metric.doneButtonBottomMargin),
+      doneButton.leadingAnchor.constraint(equalTo: self.safeAreaLayoutGuide.leadingAnchor, constant: Metric.doneButtonleadingMargin),
+      doneButton.trailingAnchor.constraint(equalTo: self.safeAreaLayoutGuide.trailingAnchor, constant: Metric.doneButtonTrailingMargin)
+    ])
+  }
+}
+
+extension UIButton {
+  func alignTextBelow(spacing: CGFloat) {
+    guard let image = self.imageView?.image else { return }
+    guard let titleLabel = self.titleLabel else { return }
+    guard let titleText = titleLabel.text else { return }
```

Comment:
```suggestion
    guard let image = self.imageView?.image,
          let titleLabel = self.titleLabel,
          let titleText = titleLabel.text else { return }
```
요렇게 써줘도 되어 보이는군요(크게 중요한건 아닙니다 ㅎㅎ)